tidb-cloud/terraform/restore-resource.md

```diff
@@ -0,0 +1,287 @@
+        id               = "780114"
+        name             = "restoreCluster"
+        project_id       = "1372813089189561287"
+        status           = "PENDING"
+    }
+    ```
+
+You can see the restore task's status is `PENDING` and the cluster's status is `INITIALIZING`.
+
+After the cluster is `AVAILABLE`, the restore task will be `RUNNING` and turn to `SUCCESS` at last.
+
+The bad news is the restored cluster is not managed by terraform. We can solve it by importing.
+
+## Import the restore cluster
+
+You can manage a cluster with Terraform by importing the cluster even if it is not created by Terraform.
+
+The following steps show you how to import the cluster created by the restore task in the last section.
+
+1. Add a cluster resource like:
+
+    ```
+    resource "tidbcloud_cluster" "restore_cluster1" {}
+    ```
+
+2. Import the cluster by `terraform import tidbcloud_cluster.restore_cluster1 projectId,clusterId`, you can get the projectId and clusterId by restore resource:
+
+    ```
+    $ terraform import tidbcloud_cluster.restore_cluster1 1372813089189561287,1379661944630264072
+    
+    tidbcloud_cluster.restore_cluster1: Importing from ID "1372813089189561287,1379661944630264072"...
+    tidbcloud_cluster.restore_cluster1: Import prepared!
+      Prepared tidbcloud_cluster for import
+    tidbcloud_cluster.restore_cluster1: Refreshing state... [id=1379661944630264072]
+    
+    Import successful!
+    
+    The resources that were imported are shown above. These resources are now in
+    your Terraform state and will henceforth be managed by Terraform.
+    ```
+
+3. Run `terraform state show tidbcloud_cluster.restore_cluster1` command to get the state of the cluster:
+
+    ```
+    $ terraform state show tidbcloud_cluster.restore_cluster1
+    
+    # tidbcloud_cluster.restore_cluster1:
+    resource "tidbcloud_cluster" "restore_cluster1" {
+        cloud_provider = "AWS"
+        cluster_type   = "DEDICATED"
+        config         = {
+            components = {
+                tidb    = {
+                    node_quantity = 2
+                    node_size     = "8C16G"
+                }
+                tiflash = {
+                    node_quantity    = 2
+                    node_size        = "8C64G"
+                    storage_size_gib = 500
+                }
+                tikv    = {
+                    node_quantity    = 6
+                    node_size        = "8C32G"
+                    storage_size_gib = 500
+                }
+            }
+            port       = 4000
+        }
+        id             = "1379661944630264072"
+        name           = "restoreCluster"
+        project_id     = "1372813089189561287"
+        region         = "eu-central-1"
+        status         = "AVAILABLE"
+    }
+    ```
+
+4. In order to manage it, you can copy it to your config file. Remember to delete the status and id, for they are computed by terraform and can not be set in the config:
```

Comment:
```suggestion
4. To manage the cluster using Terraform, you can copy the output of the previous step to your configuration file. Note that you need to delete the lines of id and status, because they will be controlled by Terraform instead:
```